runtime/src/main/cpp/Utils.h

```diff
@@ -53,3 +55,14 @@ class LockGuard {
   LockGuard(const LockGuard&) = delete;
   LockGuard& operator=(const LockGuard&) = delete;
 };
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Returns a string describing object at `address` of type `typeInfo`.
+OBJ_GETTER(DescribeObjectForDebugging, KConstNativePtr typeInfo, KConstNativePtr address);
```

Comment:
This file is not the best place for such a high-level function.
AFAIK, there is only one usage of this function in C++ code, so it is ok to move this declaration to the same file.